src/docs/Components/TextInput/code.mdx

```diff
@@ -38,6 +38,17 @@ To create an input you have one main class to apply `mc-text-input`.
 
 <Preview path="input-default" />
 
+## Variations
+
+### Available sizes
+
+You can use one of the 2 available sizes :
+
+- **small** : `mc-text-input--s`
+- **medium** : this is the default style so you don't need to add a modifier class
+
+<Preview path="input-sizes" />
```

Comment:
should we use a responsive modifier ? 
could an input be small on mobile and medium on other devices ?

Reply:
yes I think some designers will want to do something like that

Reply:
@tiloyi The small input has a height of 2mu (32px), on the preview it make 34px.
The baseline of the label is centered with the height of the input :
<img width="911" alt="Capture d’écran 2019-12-04 à 11 52 58" src="https://user-images.githubusercontent.com/43212270/70148531-a6f9ea80-16a6-11ea-8077-5ae11f7fd5ba.png">

Reply:
fixed